terracumber_config/tf_files/PR-TEST-main.tf

```diff
@@ -0,0 +1,237 @@
+terraform {
+  required_version = "1.0.10"
+  required_providers {
+    libvirt = {
+      source = "dmacvicar/libvirt"
+      version = "0.6.3"
+    }
+  }
+}
+
+
+provider "libvirt" {
+  uri = "qemu+tcp://${var.ENVIRONMENT_CONFIGURATION[var.ENVIRONMENT].hypervisor}/system"
+}
+
+module "cucumber_testsuite" {
+  source = "./modules/cucumber_testsuite"
+
+  product_version = var.PRODUCT_VERSION
+
+  // Cucumber repository configuration for the controller
+  git_username = var.GIT_USER
+  git_password = var.GIT_PASSWORD
+  git_repo     = var.CUCUMBER_GITREPO
+  branch       = var.CUCUMBER_BRANCH
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+  mirror      = var.MIRROR
+  use_mirror_images = var.USE_MIRROR
+
+  images = var.IMAGES
+
+  use_avahi    = false
+  name_prefix  = "suma-pr${var.ENVIRONMENT}-"
+  domain       = var.DOMAIN
+  from_email   = "root@suse.de"
+
+  no_auth_registry = "registry.${var.DOMAIN}"
+  auth_registry      = "registry.${var.DOMAIN}:5000/cucutest"
+  auth_registry_username = "cucutest"
+  auth_registry_password = "cucusecret"
+  git_profiles_repo = var.GIT_PROFILES_REPO
+
+  server_http_proxy = "http-proxy.${var.DOMAIN}:3128"
+  custom_download_endpoint = "ftp://${var.MIRROR}:445"
+
+  # when changing images, please also keep in mind to adjust the image matrix at the end of the README.
+  host_settings = {
+    controller = {
+      provider_settings = {
+        mac = var.ENVIRONMENT_CONFIGURATION[var.ENVIRONMENT].mac["controller"]
```

Comment:
Be caution with the default memory. Better set the same value than in CI testsuites.